.chronus/changes/named-unions-response-type-2025-10-10-16-34-2.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/http"
+  - "@typespec/openapi3"
+---
+
+support named unions in operation response types
```

Comment:
```suggestion
Support nested unions in operation return type
```

This is all this technically fix no? The fact that its named or not is just a coincidence of how you are most likely to see this?

Reply:
Yeah, you are right. I've updated the changes file and the commit message.